packages/perspective-viewer-d3fc/src/js/tooltip/selectionData.js

```diff
@@ -40,11 +40,14 @@ export function getGroupValues(data, settings) {
 
 export function getSplitValues(data, settings) {
     if (settings.splitValues.length === 0) return [];
-    const splitValues = data.key
-        ? data.key.split("|")
-        : data.mainValue.split
-        ? data.mainValue.split("|")
-        : [data.mainValue];
+    let splitValues = [data.mainValue];
+
+    if (data.key) {
+        splitValues = data.key.split("|");
+    } else if (data.mainValue?.split) {
```

Comment:
I presume this `?` is the failing logic :)

Reply:
Yup, pretty much. The default case of `[data.mainValue]` was never being reached.